### PR TITLE
swift 1.2

### PR DESCRIPTION
**Pull request by nuclearace**

### CODE REVIEW

**Comment by nuclearace** [comment]
Should be mostly socket.io-protocol compliant now.

File: SwiftIO/SocketPacket.swift [hunk 1]
```diff
@@ -217,19 +216,19 @@ class SocketPacket {
             }
         } else if let dict = data as? NSDictionary {
             var newDict = NSMutableDictionary(dictionary: dict)
-            
+
             for (key, value) in dict {
-                newDict[key as NSCopying] = _fillInPlaceholders(value)
+                newDict[key as! NSCopying] = _fillInPlaceholders(value)
```
Comment (smatte) [inline SwiftIO/SocketPacket.swift]: Using Swift 1.2 the compiler says: 'NSCopying' is ambiguous for type lookup in this context.

Comment (nuclearace) [inline SwiftIO/SocketPacket.swift]: Hm, I didn't notice that. I'll look again.

Comment (nuclearace) [inline SwiftIO/SocketPacket.swift]: I'm not getting that warning/error

**Pull request merged**